app/src/main/res/values/strings.xml

```diff
@@ -111,8 +111,8 @@
     <string name="error_client_seed_phrase_fix">Please correct the error or use another seed phrase or private key</string>
     <string name="error_client_tx_hash_validation_reason">computed safeTxHash of a transaction to confirm does not match server-returned value</string>
     <string name="error_client_tx_hash_validation_fix">Please reload the data or check that your network is secure</string>
-    <string name="error_client_unsupported_implementation_copy_reason">the implementation copy of your Safe is not supported by this app</string>
```

Comment:
was naming reverted back to master copy? What is the current situation regarding Implementation/ Master?

Reply:
The latest version on the [error doc](https://docs.google.com/document/d/1foQ9DSMgO2iN-6u4JsJIDJWB7mUHX-D8ApZWvx6oKAo/edit) says master copy: 
![image](https://user-images.githubusercontent.com/246473/110755586-672a9700-8249-11eb-87f9-06d91408392b.png)